dirty_cat/datasets/fetching.py

```diff
@@ -600,3 +604,29 @@ def fetch_drug_directory(
         },
         load_dataframe=load_dataframe,
     )
+
+
+def fetch_world_bank_data(data_code, indicator):
```

Comment:
I think we should keep the return format the same as the other fetching methods, to avoid confusion.

Reply:
As was mentionned in #303, pandas will be bumped to version 1.2.0. for the next release. This means that I can simplify this function and remove it from the fetching.py to avoid confusion.

Reply:
Thanks, done, although this format is less pertinent in this case (only 2 columns), not ready for ML.